sdcm/nemesis.py

```diff
@@ -846,6 +846,55 @@ def _add_drop_column_run_in_cycle(self):
             self._add_drop_column()
             time.sleep(1)
 
+    def choose_partitions_for_delete(self, partitions_amount, ks_cf, with_clustering_key_data=False):
```

Comment:
Maybethe strategy should be different and to make it easy for us later in case of a suspected data loss or something like that, we can always choose the same partitions.
Let's say we choose always partitions:
starting_range_without_data_validation + i * 100 + 1
So it will be something like:
1101
1201
1301
1401
...

Reply:
multiply by 100 it's so much. Did *2